ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -848,3 +848,81 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: "12h"
+  labels:
+    job-env: aws
+    job-test: e2e
+    job-release: '4.2'
+  name: release-openshift-origin-installer-e2e-aws-4.2-cnv
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-aws-cluster-profile
+      - --target=e2e-aws
+      - --template=/usr/local/e2e-aws
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+      - name: BRANCH
+        value: "4.2"
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)
+            commands: TEST_SUITE=openshift/conformance/parallel run-tests
+            openshift_installer:
+              cluster_profile: "$(CLUSTER_TYPE)"
+      - name: JOB_NAME_SAFE
+        value: e2e-aws-cnv
+      - name: TEST_COMMAND
+        value: /test.sh
+      - name: TEST_IMAGE
```

Comment:
Can I use TEST_IMAGE also this way for periodic jobs?

Reply:
nothing in your job chain is going to consume this value currently:

https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

you may need to use this template i'm working on defining here instead:
https://github.com/openshift/release/pull/4425

as it allows you to specify the name of the image you want to use for the test pod.

Reply:
> you may need to use this template i'm working on defining here instead:
> #4425

Great! :+1: This seems to be not yet ready. Is there a way on how I can help with that?

Reply:
once you have switched this PR to use  cluster-launch-installer-custom-test-image.yaml you would want to set TEST_IMAGESTREAM_TAG.

However you will also need to get your test image to be built by the CI system because TEST_IMAGESTREAM_TAG is a reference to an image that exists in the CI registry.

You can add logic to https://github.com/openshift/release/pull/4494 to get your test image built.  I will comment more there.

Reply:
Hm, since the tests live in different repos, I can probably create jobs with promotions on these repos and then reference them here?

Reply:
if the tests live in separate repos what is your plan for running all the tests together in a single job?  or do you plan to have multiple jobs?

Reply:
Hm, if one image is the way to go, I can try to squash the binaries together into one image. That should hopefully do the trick. Do the pile lines support building containers with multiple roots?

Reply:
we might be able to find a way to make it work w/ multiple images, but a single image would be easiest right now.

Reply:
and no, unfortunately I don't think we support multiple roots....  we support multistage dockerfiles, but the git cloning to munge the repos together would be something you'd have to handle which would be a bit awkward.